File: backend/connectors/memory-monitoring-connector/src/main/java/ru/nord/siwatch/backend/connectors/memorymonitoring/models/MemoryInfo.java
package ru.nord.siwatch.backend.connectors.memorymonitoring.models;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter @Setter
public class MemoryInfo {

    private String deviceId;

    private LocalDateTime deviceTime;

    private Long freeSystemMemory;

    private Long freeStorageMemory;

}

File: watch-app/SiWatchApp/Models/MemoryInfo.cs
namespace SiWatchApp.Models
{
    public struct MemoryInfo
    {
        public ulong FreeSystemMemory { get; set; }
        public ulong? FreeStorageMemory { get; set; }

        public override string ToString()
        {
            return $"MemoryInfo{{FreeSystemMemory={FreeSystemMemory},FreeStorageMemory={FreeStorageMemory}}}";
        }
    }
}
